tasks/managed/cleanup-internal-requests/cleanup-internal-requests.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: cleanup-internal-requests
+  labels:
+    app.kubernetes.io/version: "0.0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to delete internal requests associated with a pipelinerun
+  params:
+    - name: pipelineRunUid
+      type: string
+      description: The uid of the current pipelineRun. It is only available at the pipeline level
+  steps:
+    - name: cleanup-internal-requests
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+
+        if [ -n "$(params.pipelineRunUid)" ] ; then
```

Comment:
I think this will always be true now, no?

Reply:
I think a calling pipeline could pass in "".

Reply:
I guess... I don't think we protect against that in other tasks and just assume we won't do that. Oh well